docs/conf.py

```diff
@@ -48,10 +48,7 @@
 myst_heading_anchors = 3
 
 linkcheck_retries = 5
-linkcheck_ignore = [
-    r"https://doi.org/.*",
-    r"https://cernvm.cern.ch/.*"
```

Comment:
I think if you put a `,` at the end it would keep the formatting

but both looks fine for me